### ISSUE TITLE
d2l-tabs: Scrolling enhancements

### ISSUE THREAD

**Issue by tdevos-d2l** [BrightspaceUI/core#3404]
1. It would be nice if the tabs component would automatically scroll to take advantage of the full width its given when the screen resizes.

#### Daylight:
![chrome_45BfvC16TQ](https://user-images.githubusercontent.com/90270112/225350593-72f19e12-2366-4e29-99e8-0edf4b561aad.gif)

#### Consistent-Evaluation (using the Primary-Secondary template)
![chrome_dXg9r4egBB](https://user-images.githubusercontent.com/90270112/225350803-24701f4c-ac9e-4e04-bb8e-dc3716c13698.gif)

2. Might be nice and intuitive to allow swiping across tabs on mobile instead of only having the left and right arrow buttons.

#### Example use case in Consistent-Evaluation
![chrome_I5NECYyv2k](https://user-images.githubusercontent.com/90270112/225352290-1edb0288-d7ad-4565-a6ae-575c20071886.gif)

**Comment by dbatiste** [comment]
Yeah these are great suggestions. If these are improvements that are required for your tool, my recommendation is to reach out in #wc-tabs in Slack and we can loop in Nathan and Asja to prioritize.

I know the swipe functionality was discussed when this was originally built. To some extent it didn't fall into scope, but also, it was not going to be able to mimics browsers normal behaviour for swipe/flicking/acceleration, at least not without a heavy lift.
